### PR TITLE
Add BreakingT support

### PR DESCRIPTION
**Pull request by Droopy78**
Added support for the BreakingT NFTs.

### CODE REVIEW

File: src/cadence/mainnet/getNFTs.cdc [hunk 1]
```diff
@@ -2429,8 +2433,8 @@ pub fun getAmericanAirlinesNFT(owner: PublicAccount, id: UInt64): NFTData? {
             NFTMedia(uri: setMeta!["preview"], mimetype: "image")
         ],
         metadata: {
-            "edition": nft!.editionNum.toString(),
-            "max_editions": nftEditions!.toString(),
+            "editionNumber": nft!.editionNum.toString(),
```
Comment (niveda-krish) [inline src/cadence/mainnet/getNFTs.cdc:2436]: Did you intend to make this rename change?

**Comment by niveda-krish** [comment]
Could you also resolve merge conflicts? Our release is this Thursday, would be great to get your changes in before that

File: src/cadence/mainnet/getNFTs.cdc (hunk 1 above)
Comment (Droopy78) [reply src/cadence/mainnet/getNFTs.cdc:2436]: Yes the name for this was changed throughout the file based on @albeethekid recommendation to align with the Blocto naming convention

**Comment by Droopy78** [comment]
@niveda-krish All is merged and should be ready.

**Review by niveda-krish — APPROVED** [review]

**Pull request merged**